app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -381,10 +384,20 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, final int pos
                             .setOnClickListener(new View.OnClickListener() {
                                 @Override
                                 public void onClick(View view) {
-                                    Intent i = new Intent(Intent.ACTION_VIEW);
-                                    i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                                    i.setData(Uri.parse(url));
-                                    zulipApp.startActivity(i);
+                                    if (Build.VERSION.SDK_INT < 15)
+                                    {
+                                        Intent i = new Intent(Intent.ACTION_VIEW);
+                                        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                                        i.setData(Uri.parse(url));
+                                        zulipApp.startActivity(i);
+                                    }
```

Comment:
At first a i added that but it was working fine in both the versions so i just removed that. 
Well added it again. Review now. :)